user/mongoStoreClient.go

```diff
@@ -38,6 +38,17 @@ func NewStore(config *goComMgo.Config, logger *log.Logger) (*Client, error) {
 				SetUnique(false).
 				SetExpireAfterSeconds(0),
 		}},
+		USERS_COLLECTION: {{
+			Keys: bson.D{{Key: "userid", Value: 1}},
+			Options: options.Index().
+				SetName("UserId").
+				SetUnique(true),
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏i think the unique requirement is not needed for these indexes